themes/bootstrap5/templates/hierarchy/tree.phtml

```diff
@@ -29,15 +29,20 @@
     // Use escapeHtml for performance (this data shouldn't contain anything that escapeHtmlAttr would be useful with)
     $escapeAttr = $escape;
     $truncate = $this->plugin('truncate');
+    $isOneNode = count($nodes) === 1;
     foreach ($nodes as $node) {
       $hasChildren = !empty($node->children);
       $liClasses = [];
+      $openTopLevel = false;
       if ($hasChildren) {
         $liClasses[] = 'hierarchy-tree__parent';
       }
       if ('collection' === $node->type) {
         $liClasses[] = 'hierarchy-tree__collection';
         $icon = 'hierarchy-collection';
+        if ($nodeId === 0) {
```

Comment:
Should we also check that `$parentNodeId` is empty here if we only want to impact the very top of the tree? It seems to be working fine for me as-is in my test environment, but that data is very simple... I'm not too familiar with the code (so @EreMaijala may have a better-informed opinion), but since there is recursion here, I thought there might be reason to add an extra restriction.